core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/LambdaVariables/LambdaVariables.js

```diff
@@ -119,19 +140,126 @@ function VariableValue({ variable }) {
   return value;
 }
 
+function SecretVariableValue({ variable }) {
+  const [show, setShow] = useState(false);
+
+  const { data, loading, error } = useQuery(GET_SECRET, {
+    variables: { namespace: variable.namespace, name: variable.resourceName },
+  });
+
+  if (loading) return null;
```

Comment:
```suggestion
  if (loading){
      return null;
  }    
```

in other places add `{` and `}` too